packages/core/test/Asset/util/isValidHandle.test.ts

```diff
@@ -23,4 +26,10 @@ describe('isValidHandle', () => {
   test.each(invalidHandles)('returns false for invalid handle %s', (handle) => {
     expect(Asset.util.isValidHandle(handle)).toBe(false);
   });
+  test.each(invalidHandles)('returns false for invalid handle with spaces and tabs', (handle) => {
+    expect(Asset.util.isValidHandle(handle)).toBe(false);
```

Comment:
These new tests seem to be duplicating the original tests. Is it not enough to add items to `validHandles` and `invalidHandles`? :thinking:

Reply:
addressed: 8890ccc43abe351972e53dcb5d37ce31744d329f